### ISSUE TITLE
local/train_audiosearch.py

### ISSUE THREAD

**Issue by kinjaljain** [saikrishnarallabandi/interspeech-2020-papers#4]
Train Accuracy in 1000s and test accuracy a little more than 100%

**Comment by kinjaljain** [comment]
@saikrishnarallabandi

**Comment by saikrishnarallabandi** [comment]
This line seems strange:
accuracy = 100 * correct_train / (len(train_loader) * search.shape[0])

The last batch will always have less number of examples and hence search.shape[0] will be a wrong thing to use here. 

May be try using classificaton_accuracy from sklearn

**Comment by saikrishnarallabandi** [comment]
closing this @kinjaljain 
reopen if persists
